docs/integration/KafkaIntegration.md

```diff
@@ -2,12 +2,12 @@
 sidebar_position: 1
 ---
 
-# Kafka
+# Kafka | Streaming only
```

Comment:
I'm not sure wheather we can have this note here, node docs links may stop working

Reply:
My Visual Code interpreted this link correctly, not sure about Node / Docusaurs.
Do you mean "_existing incoming links will stop working_" or do you mean "_this may not work with Docusaurus"_?

Reply:
everything's good 👍